iree/test/e2e/regression/generate_e2e_matmul_tests.py

```diff
@@ -454,6 +588,13 @@ def parse_arguments():
                       choices=[s.value for s in ShapesId],
                       help="Collection of matrix shapes to test",
                       required=True)
+  parser.add_argument("--compilation_info",
+                      type=str,
+                      choices=[i.value for i in CompilationInfoId],
+                      help="Collection of compilation info setups to test",
+                      default="none",
```

Comment:
Wouldn't it be more usual to have the default value be the empty string `""` , rather than a custom keyword `"none"` ? That would also allow writing more idiomatic code in my suggestion below, `if not compilation_info_id:`.

Reply:
Done. however note that the compilation_info_id is an enum so I must check for an explicit value